### DISCUSSION TITLE
Can Pixi3d be used on the server-side?

### DISCUSSION THREAD

**Discussion by DOEHOONLEE** [jnsmalm/pixi3d#133] [Q&A]
Hi~!
I was wondering if I can use pixi3d on the server-side, not on the front-end side. Is this possible?

**Comment by jnsmalm** [comment]
Hi! It should work together with pixi.js-node after I have added support for PixiJS v7, this is still under development though.

**Reply by goldenratio** [reply]
offtopic: does webgl work in node? can it make use of GPU. Wondering If web based game streaming is possible.

**Reply by jnsmalm** [reply]
https://www.npmjs.com/package/@pixi/node works in node and uses WebGL. Not sure about web-based streaming though.
